app/views/shares/index.html.erb

```diff
@@ -3,13 +3,26 @@
 <div class="col-xs-10 col-xs-offset-1 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3">
   <% @shares.each do |share| %>
   <div class="booyah-box post">
-    <div class="handle"><%= current_user.name %></div>
-    <%= link_to share.title, share_path(share) %>
-    <p>
-      <%= linkify_hashtags(share.body) %>
-    </p>
-    <%= pluralize(share.votes.count, "votes") %>
-    
+    <div class="handle">
+      <h4><%= link_to share.title, share_path(share) %></h4>
+      <p><%= linkify_hashtags(share.body) %></p>
+      <small><%= share.user.name  %></small>
+      <p><%= pluralize(share.votes.count, "votes") %></p>
+    </div>
+
+    <br />
+
+    <% if current_user == share.user %>
+      <div>
+        <div class="btn btn-danger">
+          <%= link_to 'Delete', share_path(share), method: :delete, data: {confirm: 'Are you sure?'} %>
```

Comment:
Jus realized that. I was not totally sure what the app did at the time. There are code shares and comments. I will update the code.